ndcube/ndcube.py

```diff
@@ -174,9 +206,11 @@ class NDCubeBase(NDCubeSlicingMixin, NDCubeABC):
         Default is False.
 
     """
+    _extra_coords = NDCubeLinkedDescriptor(ExtraCoords)
```

Comment:
Add comment here like, `# Initializing empty ExtraCoords and GlobalCoords instances on NDCube.`

By the by, why are these lines floating on their own and not in `__init__`.  Is it for the same reason as in astropy modeling (whatever that reason actually is?)

Reply:
These are class level attributes not instance level attributes (as they would be if they were `self.whatever` in the constructor). All instances of this class share the one instance of this descriptor object.